src/index.html

```diff
@@ -14,37 +14,39 @@
 </head>
 
 <body>
-  <header class="o-header">
+  <header class="c-header">
     <!-- Brand -->
-    <div class="c-brand-logo">
+    <span class="c-brand-logo">
       <img src="assets/Logo@2x.png" alt="bork co.">
-    </div>
+    </span>
 
     <!-- Mobile Navigation -->
-    <div class="o-nav__mobile">
+    <nav class="o-nav__mobile">
```

Comment:
I feel like navigation might make more sense as a component rather than an object. What factors led to you settling upon object?

Reply:
Good catch, I haven't really changed various objects from the original iteration to components yet but have been meaning to.

Reply:
This has been changed to c-nav__mobile as well as the other to c-nav__desktop